akashian/phase1.md

```diff
@@ -21,6 +21,15 @@ Please store the menonic keys in a safe place
 
 {% endhint %}
 
+**Note: If joining the network after Genesis files have been finalized; your keys must be initialized on the chain by having funds transfered to your address.**
+
+```sh
+akashctl keys show $TEAM -a
+<Address to request faucet funds>
+```
+
+Join the [Akash Network](https://riot.im/app/#/room/#akashnet:matrix.org) Riot channel and request funds to be transfered from an admin.
+
```

Comment:
This section looks duplicate. Everyone has to create the keys irrespective of joining time. So may be we can just show a link here. something like, If you don't have a key yet, create your keys (link)

Reply:
Yeah, I agree. I'm working on writing a `guide` doc for keys.